base/complex.jl

```diff
@@ -1116,7 +1110,23 @@ big(::Type{Complex{T}}) where {T<:Real} = Complex{big(T)}
 big(z::Complex{T}) where {T<:Real} = Complex{big(T)}(z)
 
 ## Array operations on complex numbers ##
+"""
+    complex(A::AbstractArray)
+
+Return an array containing the complex analog of each entry in array `A`.
+
+Equivalent to `complex.(A)`, except that the return value may alias all or part
```

Comment:
Some users may not know what `alias` means, would it be worth having something like
```julia
julia> y = rand(Complex{Float64}, 3);

julia> complex(y) === y
true

julia> complex.(y) === y
false
```

Reply:
Or write `may alias (return the same array for) all or part`?

Reply:
adding the doctest looks like a good idea.

regarding the `alias` language, I agree with the suggestions that it can be made more new-user-friendly, however, I should note that I chose this wording to match that of `convert(T, x)`, which is written

```
If `T` is a collection type and `x` a collection, the result of
`convert(T, x)` may alias all or part of `x`.
```jldoctest
julia> x = Int[1, 2, 3];

julia> y = convert(Vector{Int}, x);

julia> y === x
true
```

is it ok if the two wordings do not match, or should I change both?

Reply:
Yes, updating both to have the same parenthetical seems like a good change to make here

Reply:
updated